gradle/libs.versions.toml

```diff
@@ -7,6 +7,7 @@ androidxAppCompat = "1.5.1"
 androidxBrowser = "1.4.0"
 androidxComposeBom = "2023.06.01"
 androidxComposeCompiler = "1.5.0"
+androidxComposeRuntime = "1.5.2"
```

Comment:
Since #947 has been merged, this is no longer necessary and should be removed.
Compose BoM `2023.09.00` already depends on runtime `1.5.1` (this feature was added in 1.5.0).

Reply:
@SimonMarquis thank you for pointing that out, updated now.